Recipe/recipe.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title> My Recipe Page</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Sarah's Recipe Page</title>
+  <link href="./recipe.css" rel="stylesheet" type="text/css" />
+  
+
+
+  <head>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Teko|Abril Fatface|Yeseva One|Oleo Script|Tangerine|Assistant">
+    <style>
+    h1 {font-family: "Oleo Script", serif; font-size: 62px; text-shadow: 4px 4px 4px rgb(124, 152, 123); color: rgb(34, 77, 31)}
+    h2 {font-family: "Assistant", serif; font-size: 25px; color: rgb(0, 0, 0)}
+    </style>
```

Comment:
I would only suggest to use a `<style>` tag if you hadn't learned about linking a CSS file but you already know how to do that... So move these CSS rules to your CSS file, they serve the exact same purpose.